v1.5.1.md

```diff
@@ -6,6 +6,12 @@ The Harvester team appreciates your contributions and looks forward to receiving
 
 > [!CAUTION]
 >
+> A [Longhorn issue](https://github.com/longhorn/longhorn/issues/11158) causes virtual machines that use migratable [RWX volumes](https://docs.harvesterhci.io/v1.5/rancher/csi-driver/#rwx-volumes-support) to restart unexpectedly when the CSI plugin pods are restarted. The issue affects Harvester v1.4.x, v1.5.0, and v1.5.1.
+>
+> You can work around it by disabling the **Storage Network for RWX Volume Enabled** setting on the [embedded Longhorn UI](https://docs.harvesterhci.io/v1.5/troubleshooting/harvester/#access-embedded-rancher-and-longhorn-dashboards). The issue will be fixed in Longhorn {versions}, which will be included in Harvester {versions}. For more information, see GitHub issues [#8534](https://github.com/harvester/harvester/issues/8534) and [#11158](https://github.com/longhorn/longhorn/issues/11158).
```

Comment:
Fixed on Longhorn v1.8.3, v1.9.1, and later versions.
v1.9.1 will be included in Harvester v1.6.0